node/src/actors/chain_manager/mod.rs

```diff
@@ -807,15 +808,19 @@ fn update_reputation(
 }
 
 fn show_info_tally(tally_tx: &TallyTransaction, block_epoch: Epoch) {
```

Comment:
Sorry for bothering, let's take the chance to rename this:
```suggestion
fn show_tally_info(tally_tx: &TallyTransaction, block_epoch: Epoch) {
```